packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -270,7 +299,14 @@ export class ChildLogger extends TelemetryLogger {
     public static create(
         baseLogger?: ITelemetryBaseLogger,
         namespace?: string,
-        properties?: ITelemetryLoggerPropertyBags): TelemetryLogger {
+        properties: ITelemetryLoggerPropertyBags = {},
+        globalProperties: ITelemetryLoggerPropertyBags = {},
+        layerVersion?: string,
```

Comment:
Thinking about ditching this generic capability and just have loader and driver pass `fooVersion` as a global property.  Thoughts?